Gopkg.toml

```diff
@@ -1,4 +1,5 @@
 required = [
+  "github.com/golang/protobuf/descriptor",
```

Comment:
You shouldn't need to explicitly add this as we're importing it.

Reply:
you would think wouldn't you:

`"github.com/golang/protobuf/descriptor" is not imported by your project, and has been temporarily added to Gopkg.lock and vendor/.
If you run "dep ensure" again before actually importing it, it will disappear from Gopkg.lock and vendor/.`

Reply:
That doesn't sound right at all

Reply:
agreed, maybe it's because there are no go files in the root of the repo? dep is weird sometimes...

Reply:
I don't think that should matter. I've never had to add a package explicitly like this. Let me try checking out this branch.

Reply:
huh you're right, if I get rid of the requires everything still works fine, I guess I just confused dep by doing `dep ensure -add github.com/golang/protobuf/descriptor`